code/modules/mob/living/living.dm

```diff
@@ -146,7 +146,7 @@ default behaviour is:
 			..()
 			if (!istype(AM, /atom/movable) || AM.anchored)
 				//VOREStation Edit - object-specific proc for running into things
-				if((confused || is_blind()) && stat == CONSCIOUS && prob(50) && m_intent=="run")
+				if(((confused || is_blind()) && stat == CONSCIOUS && prob(50) && m_intent=="run") || flying)
```

Comment:
If I'm reading this correctly, walking into any anchored object while flying will 100% result in stumbling? Is that correct?

Reply:
Yes. If they bump into anything at all, they immediately "stumble" (see: crash) into it.